examples/decoding/plot_decoding_xdawn_eeg.py

```diff
@@ -62,29 +62,30 @@
                 add_eeg_ref=False, verbose=False)
 
 # Create classification pipeline
-X = EpochsVectorizer().fit_transform(epochs)
-labels = epochs.events[:, -1]
-clf = make_pipeline(XdawnTransformer(n_components=3,
-                    n_chan=epochs.info['nchan']),
+clf = make_pipeline(Xdawn(n_components=3),
+                    EpochsVectorizer(),
                     MinMaxScaler(),
                     LogisticRegression(penalty='l1'))
 
+# Get the labels
+labels = epochs.events[:, -1]
+
 # Cross validator
 cv = StratifiedKFold(y=labels, n_folds=10, shuffle=True, random_state=42)
 
 # Do cross-validation
-y_pred = np.empty(len(labels))
+preds = np.empty(len(labels))
```

Comment:
I have changed it back to the original. Are you sure this should be changed here and not a new PR?